docs/examples/legacy/Legacy.ipynb

```diff
@@ -0,0 +1,127 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction\n",
```

Comment:
will this become the title of the page in the docs and in the table of contents? perhaps we need to make it less generic:
```suggestion
    "# Legacy features\n",
```